src/main/kotlin/no/nav/syfo/tilgang/TilgangService.kt

```diff
@@ -264,14 +264,15 @@ class TilgangService(
         callId: String,
         appName: String,
         doAuditLog: Boolean = true,
+        bulk: Boolean = false,
     ): Deferred<Tilgang> =
         coroutineScope.async {
             val veilederIdent = token.getNAVIdent()
             val cacheKey = "$TILGANG_TIL_PERSON_PREFIX$veilederIdent-$personident"
             val cachedTilgang: Tilgang? = valkeyStore.getObject(key = cacheKey)
 
             val tilgang = cachedTilgang ?: checkTilgangToPersonAndCache(callId, token, personident, cacheKey)
-            if (cachedTilgang == null) {
+            if (cachedTilgang == null && !bulk) {
```

Comment:
Ønsker ikke å sjekke mot tilgangsmaskin her i tillegg for de kallene som kommer fra vår egen bulk-håndtering.